test/test_mkldnn.py

```diff
@@ -97,6 +97,17 @@ def test_conversion(self):
                                        "Cannot access data pointer of Tensor that doesn't have storage",
                                        lambda: mkldnn_tensor.data_ptr() != 0)
 
+    def test_copy(self):
+        x = torch.randn(4, 5, dtype=torch.float32)
+        mkldnn_x = x.to_mkldnn()
```

Comment:
what happens if one of the arguments is non-mkldnn tensor? Will it still be sent to mkldnn dispatch?
Can you please add testcases for that?